task/sast-shell-check/0.1/sast-shell-check.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: sast-shell-check
+spec:
+  description: >-
+    ShellCheck is a static analysis tool, gives warnings and suggestions for bash/sh shell scripts.
+  results:
+    - description: Tekton task test output.
+      name: TEST_OUTPUT
+  params:
+    - name: image-url
+      description: Image URL.
+      type: string
+      default: ""
+    - name: image-digest
+      description: Image digest to report findings for.
+      type: string
+      default: ""
+    - name: KFP_GIT_URL
+      description: git repository to download known false positives files from
+      type: string
+      default: ""
```

Comment:
Please insert the following comment so that we remember to override the default once Konflux is ready:
```
# FIXME: Red Hat internal projects will default to https://gitlab.cee.redhat.com/osh/known-false-positives.git when KONFLUX-4530 is resolved
```

Reply:
Fixed in https://github.com/konflux-ci/build-definitions/pull/1437/commits/1c1285c904b0a45d8a524b47dfabb58f53abeb00